airflow/providers/apache/spark/hooks/spark_jdbc.py

```diff
@@ -196,6 +196,7 @@ def _resolve_jdbc_connection(self) -> Dict[str, Any]:
             conn_data['password'] = conn.password
             extra = conn.extra_dejson
             conn_data['conn_prefix'] = extra.get('conn_prefix', '')
+            conn_data['conn_suffix'] = extra.get('conn_suffix', '')
```

Comment:
What is this for?

Reply:
It's just a proposal. For example, I have Spark JDBC connection for MS SQL Server, in which I have to specify the database' name to connect to:

`jdbc:sqlserver://hostname:port;databaseName=db;`

The only way to do this is to put the entire URL-string in the `Host` field of the 'Edit connection' form.

So, it seems useful to add a suffix to deal with cases like this.